package.json

```diff
@@ -1,53 +1,51 @@
 {
-  "name": "beanstalkify",
+  "name": "elasticsprout",
```

Comment:
What's the reason to change the name here?

Reply:
Apologies @liamqma that was my bad. I was testing this project and forgot to change the name back. I've made a commit to rename it back to what it was.